apps/03TicTacToe.js

```diff
@@ -22,23 +28,37 @@ function printBoard() {
 }
 
 function horizontalWin() {
-    // Your code here
+    return (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn)
+    || (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn)
+    || (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)
+
 }
 
 function verticalWin() {
-    // Your code here
+    return (board[0][0] === playerTurn && board[1][0]=== playerTurn && board[2][0] === playerTurn)
+    || (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn)
+    || (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] ===playerTurn)
 }
 
 function diagonalWin() {
-    // Your code here
+  return (board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn)
+    || (board[0][2] === playerTurn && board [1][1] === playerTurn && board [2][0] === playerTurn)
 }
 
 function checkForWin() {
-    // Your code here
+      if(verticalWin() === true ||horizontalWin() === true||diagonalWin() === true) {
+        console.log('Player' + playerTurn + 'won!');
+        return true;
+      }else{
+        console.log("Take another turn.");
+        return false;
+      }
 }
 
 function ticTacToe(row, column) {
-    // Your code here
+    board[row][column]= playerTurn;
+    checkForWin();
```

Comment:
So, you're checking checkForWin here, but what's going to happen if checkForWin is true/false? Maybe using a condition will put your checkForWin to use, something like:

if(checkForWin()){
// do something if a player won;
}